MantidPlot/src/Graph.cpp

```diff
@@ -4345,8 +4345,10 @@ void Graph::copy(Graph *g) {
       if (pie)
         pie->addLabel(dynamic_cast<PieLabel *>(t), true);
       else
+        // cppcheck-suppress leakReturnValNotUsed
         insertText(t);
     } else
+      // cppcheck-suppress leakReturnValNotUsed
       insertText(t);
```

Comment:
The plot itself,

here's the definition of that fn

LegendWidget *Graph::insertText(LegendWidget *t) {
  LegendWidget *aux = new LegendWidget(d_plot);
  aux->clone(t);
  return aux;
}

The new legend widget is owned by d_plot